Lib/importlib/abc.py

```diff
@@ -367,13 +367,19 @@ def iterdir(self):
     def read_bytes(self):
         """
         Read contents of self as bytes
+
+        If the file cannot be found, FileNotFoundError should be raised,
+        although this is not guaranteed for compatibility reasons.
         """
         with self.open('rb') as strm:
             return strm.read()
 
     def read_text(self, encoding=None):
         """
         Read contents of self as text
+
+        If the file cannot be found, FileNotFoundError should be raised,
+        although this is not guaranteed for compatibility reasons.
```

Comment:
Since these implementations don't perform any specific checks on fileness, I'd leave out any documentation about what exceptions are raised and defer to the docs for open.